tests/unit/chrome_client_test.ts

```diff
@@ -396,6 +397,84 @@ Rhum.testPlan("tests/unit/chrome_client_test.ts", () => {
     });
   });
 
+  Rhum.testSuite("takeScreenshot()", () => {
+    const ScreenshotsFolder = "./ScreenshotsChrome";
+    Rhum.beforeAll(() => {
+      try {
+        Deno.removeSync(ScreenshotsFolder, { recursive: true });
+      } catch (e) {
+        console.debug((e as Error).message);
```

Comment:
should need the cast here right? `message` is already a property on `Error` which `e` is?

Reply:
oh ok, gotcha, will resolve this